### ISSUE TITLE
deADBolt hangs on main menu

### ISSUE THREAD

**Issue by thesle3p** [nathanakalish/deADBolt#2]
So far deADBolt does not work out of the box on the most recent version of nethunter or through the kali main menu, when I try and run it via sh ./main.sh it shows the main menu for deADBolt and then says main.sh[184]: read -p no coprocess and does not respond to option selections. Also if deADBolt is selected  from the kali nethunter menu it loops back to the USB attack options menu. This bug persists through multiple flashes on Nethunter.

**Comment by nathanakalish** [comment]
Yeah, it is because deADBolt isn't actually complete... I need to fix it, but one of the parts I need doesn't actually exist yet!
